### DISCUSSION TITLE
Just seeing spinnig wheels for 1 page on the details  dashboard

### DISCUSSION THREAD

**Discussion by a4z** [plausible/analytics#1264] [Anything Else]
I have 1 page in my dashboard that does not show details.

What could be the reason for that ? Page is up for more than 12 hours now, 
There was 1 special thing, the page is 2 times in the list, since one shared url has a // at the begin of the page, instead of a /. 
Could that cause confusion for Plausible? 

<img width="1094" alt="Screenshot 2021-08-22 at 22 36 03" src="https://user-images.githubusercontent.com/6172295/130369340-9058538e-e768-4dba-ae61-4e7990427b72.png">

**Comment by a4z** [comment]
It's all my fault ... sorry for the wrong alarm

But maybe still a valuable info for others:

I named the blog page `From google analytics to Plausible` and have therefore an URL https://a4z.gitlab.io/blog/2021/08/22/From-google-analytics-to-plausible.html , and since this url contains `google-analytics`,  browsers with tracking protection (I have firefox and vivaldi) block everything containing that part in an URL....

I noticed that since also the discus comment section did not load on the blog page.
So I tested private mode and other browsers, and there everything works.

So Plausible works fine!

Fazit: Pay attention how you name your URLs, if they contain some keywords, they might be filtered out by browsers with some in built protection

**Reply by ukutaht** [reply]
Wow, that's some pretty aggressive blocking. Can imagine a lot of legit websites/pages with `google-analytics` in the URL :)

Glad you got the issue fixed!
